sound/soc/sof/Makefile

```diff
@@ -2,7 +2,7 @@
 
 snd-sof-objs := core.o ops.o loader.o ipc.o pcm.o pm.o debug.o topology.o\
 		control.o trace.o utils.o sof-audio.o
-snd-sof-$(CONFIG_SND_SOC_SOF_DEBUG_PROBES) += probe.o compress.o
+snd-sof-$(CONFIG_SND_SOC_SOF_DEBUG_PROBES) += probe.o probe_compress.o
```

Comment:
should we rename the compress.h to probe_compress.h as well?
I contains only sof_probe_compr* prefixed definitions and a declaration.

Reply:
@ujfalusi yes. will resend today with your comments addressed.

Reply:
@ujfalusi Done!